packages/react/src/JsonForms.tsx

```diff
@@ -217,6 +218,8 @@ export const JsonForms = (
     refParserOptions,
     onChange
   } = props;
+  const uischemaToUse =
+    uischema !== undefined ? uischema : Generate.uiSchema(schema);
```

Comment:
Minor possible improvement: I think we could make this check stricter by switching the logic and test for `typeof usichema === 'object'`.

Reply:
done